registration_ref/app.py

```diff
@@ -40,58 +50,71 @@ def create_in_foundries(client_cert: str, api_token: str, name: Optional[str] =
             "https://api.foundries.io/ota/devices/", headers=headers, json=data
         )
         if r.status_code == 409:
+            log.error("Device creation conflict detected: %s", r.text)
             abort(409, description=r.text)
         if r.ok:
+            log.info("Device successfully created in foundries")
             return
-        msg = f"Unable to create device on server: HTTP_{r.status_code} - {r.text}"
-        app.logger.error(msg)
+        msg = "Unable to create device on server: HTTP_%s - %s" % (r.status_code, r.text)
+        log.error(msg)
```

Comment:
Just a small nit.

I think you wanted to do `log.error("Unable to create device on server: HTTP_%s - %s", r.status_code, r.text)` here.